PROTOCOL.md

```diff
@@ -763,10 +769,10 @@ In `id ` mode, readers must resolve columns by using the `field_id` in the parqu
 In `name` mode, readers must resolve columns in the data files by their physical names as given by the column metadata property `delta.columnMapping.physicalName` in the Delta schema. Partition values and column level statistics will also be resolved by their physical names. For columns that are not found in the files, `null`s need to be returned. Column ids are not used in this mode for resolution purposes.
 
 # Deletion Vectors
-Enablement:
- - To enable Deletion Vectors on a table, the table must have Reader Version 3 and Writer Version 7. A feature name `deletionVectors` must exist in the table's `readerFeatures` and `writerFeatures`.
+To support this feature:
+ - To support Deletion Vectors, a table must have Reader Version 3 and Writer Version 7. A feature name `deletionVectors` must exist in the table's `readerFeatures` and `writerFeatures`.
 
-When enabled:
+When supported:
```

Comment:
I think there's something missing in the protocol about _activating_ or _enabling_ DVs. AFAICT there is no mention of `delta.enableDeletionVectors`. Can you fix that as well?

Reply:
Done!